docs/source/index.rst

```diff
@@ -1,19 +1,54 @@
 Welcome to MyoSuite's documentation!
 =====================================
 
-`MyoSuite <https://sites.google.com/view/myosuite>`_  is a collection of musculoskeletal environments and tasks simulated with the `MuJoCo <http://www.mujoco.org/>`_ physics engine and wrapped in the OpenAI ``gym`` API to enable the application of Machine Learning to bio-mechanic control problems
+`MyoSuite <https://sites.google.com/view/myosuite>`_  is a collection of musculoskeletal environments and tasks simulated with the `MuJoCo <http://www.mujoco.org/>`_ physics engine and wrapped in the OpenAI ``gym`` API to enable the application of Machine Learning to bio-mechanic control problems.
+
+Check our `github repository <https://github.com/MyoHub/myosuite>`__ for more technical details.
+
+Our paper can be found at: `https://arxiv.org/abs/2205.13600 <https://arxiv.org/abs/2205.13600>`__
+
+Before you start, we highly recommended to have a basic understanding of `OpenAI Gym API <https://gymnasium.farama.org/>`__. Principle knowledge in Reinforcement Learning is also expected for a best experience.
+
 .. note::
 
    This project is under active development.
 
+
+Main features
+============================================
+   * Bio-mechanic based simulation of musculoskeletal
+   * Task-driven controller for motion synthesis
+
+.. TODO: confirm features for myosuite
```

Comment:
@raku-slyu - what's this ToDo?

Reply:
It's meant to include main features of myosuite. Inspiration comes from the doc of stable_baseline3 and Jax

Reply:
parts removed

Reply:
update on commit : [https://github.com/MyoHub/myosuite/pull/192/commits/bdb4236a3dd3560280f1247c874abdab876d76a9](url)